R/pkg/R/functions.R

```diff
@@ -41,14 +41,21 @@ NULL
 #' @param x Column to compute on. In \code{shiftLeft}, \code{shiftRight} and \code{shiftRightUnsigned},
 #'          this is the number of bits to shift.
 #' @param y Column to compute on.
-#' @param ... additional argument(s). For example, it could be used to pass additional Columns.
```

Comment:
Right, `...` is not used in any of these functions here. 
But it is still documented because one of the generic methods `bround` has it.
![image](https://user-images.githubusercontent.com/11082368/27622442-00d5824a-5b8c-11e7-9675-53a717c90075.png)